ansible/group_vars/controllers

```diff
@@ -0,0 +1,16 @@
+---
+# FIXME(w-miller): set this more cleverly?
+base_path: "~/tenks/"
+allocations_file_path: "{{ base_path + '/allocations.yml' }}"
+
+flavours:
+  flav0:
+    memory_mb: 4096
+    vcpus: 4
+    volumes:
+      - capacity: 10GB
+    physical_networks:
+      - physnet1
+
+specs:
+  flav0: 4
```

Comment:
Don't these variables apply more to hypervisors?

Reply:
Makes sense now I know that controllers == localhost

Reply:
Should this file be `group_vars/localhost`, then?

Reply:
Or host_vars/localhost